app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!, :set_item
+
+  def create
+    @comment = @item.comments.build comment_params.merge(user: current_user)
+    if @comment.save
+      redirect_to @item, notice: t('.created')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.